src/styles/stars.css

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-image: url(../images/star.svg);
+  background-position: 50%;
+}
+
+.stars--4 > .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
You still have only one option for active stars

It should have functionality from previous task, if we set other modifier, for example stars--3, it should show 3 active stars, but now it will just disable active stars

Reply:
I've written about it during previous review, so please be careful next time